roboteam_ai/src/utilities/GameStateManager.cpp

```diff
@@ -13,7 +13,21 @@ roboteam_msgs::RefereeData GameStateManager::getRefereeData() {
 
 void GameStateManager::setRefereeData(roboteam_msgs::RefereeData refMsg) {
     GameStateManager::refMsg = refMsg;
-    strategymanager.setCurrentRefGameState(static_cast<RefCommand>(refMsg.command.command));
+    auto cmd = static_cast<RefCommand>(refMsg.command.command);
```

Comment:
this should happen in strategymanager, not in gamestate manager, for consistency.

Reply:
What exactly are you referring to with 'this' here? The static casting? I do not follow you